Amazon.QLDB.Driver/Amazon.QLDB.Driver.csproj

```diff
@@ -56,6 +56,9 @@
     <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
       <_Parameter1>$(AssemblyName).Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100516caa6311db961cb07702d63876c5b8cbd661557cd18bde9fb966bb30a4442abb27a4a5aca5af15c97b77f3f8f683eb1ca32cd7e8d1edcbbb1a62fe215001d507c2437f052b29540b7a11edbdc7dfe12de00c37f9e70c7e85a04541858ca46bb2581099780121ee8041732b7214ec9b5c483ef13c1db6d5f86a71fcc014dcaf</_Parameter1>
     </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>Amazon.QLDB.Driver.Tests.Mac, PublicKey=0024000004800000940000000602000000240000525341310004000001000100516caa6311db961cb07702d63876c5b8cbd661557cd18bde9fb966bb30a4442abb27a4a5aca5af15c97b77f3f8f683eb1ca32cd7e8d1edcbbb1a62fe215001d507c2437f052b29540b7a11edbdc7dfe12de00c37f9e70c7e85a04541858ca46bb2581099780121ee8041732b7214ec9b5c483ef13c1db6d5f86a71fcc014dcaf</_Parameter1>
```

Comment:
Hardcoding the assembly name because the assembly name is derived from the solution file name: `Amazon.QLDB.Driver.Mac` which doesn't correspond to `Amazon.QLDB.Driver.Tests.Mac`